.github/workflows/macos.yml

```diff
@@ -1,8 +1,18 @@
 name: macos
 
 on:
-  push:
+  workflow_dispatch:
   pull_request:
+  push:
+    branches:
+      - "main"
+
+permissions:
+  contents: read
+
+env:
+  GCS_ACCOUNT_SECRET: 'secret/observability-team/ci/snapshoty'
+  DOCKER_REGISTRY_SECRET: 'secret/observability-team/ci/docker-registry/prod'
```

Comment:
I cannot find where these env variables are used, are they leftovers?

Reply:
👍 The change was in the stash...